api/tests/integration/infrastructure/utils/xml/siecle-file-streamer_test.js

```diff
@@ -34,6 +34,27 @@ describe('SiecleFileStreamer', function () {
           expect(error).to.be.an.instanceof(FileValidationError);
           expect(error.code).to.equal('INVALID_FILE');
         });
+
+        it('log only the first sax error', async function () {
```

Comment:
`nitpick`
On est pas assuré que ce soit la premère, et ce n'est pas important
```suggestion
        it('log only one sax error', async function () {
```